ui/app/adapters/pki/key.js

```diff
@@ -4,12 +4,12 @@ import { encodePath } from 'vault/utils/path-encoding-helpers';
 export default class PkiKeyAdapter extends ApplicationAdapter {
   namespace = 'v1';
 
-  optionsForQuery(id) {
```

Comment:
options didn't seem necessary because the urls are different to `GET` a single key vs. `LIST` all keys